Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -145,6 +149,8 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 
 	void SetLifetime(int32 Lifetime);
 
+	void SetStatStartStopTime(int32 StartDelayTime, int32 StopDelayTime);
```

Comment:
I'm not sure this is necessary, if stop time is enabled when the start event occurs then it's okay to be relative...

Reply:
We call `SetStatStartStopTime` twice, one for WorkerFlag and one for Commandline, So I prefer making it as a function.